frontend/app/components/person-form.js

```diff
@@ -0,0 +1,165 @@
+import BaseFormComponent from "./base-form-component";
+import { inject as service } from "@ember/service";
+import { action, computed } from "@ember/object";
+import { isBlank } from "@ember/utils";
+import { getNames as countryNames } from "ember-i18n-iso-countries";
+import Person from "../models/person";
+import { addObserver } from "@ember/object/observers";
+
+export default class PersonFormComponent extends BaseFormComponent {
+  @service store;
+  @service session;
+  @service ajax;
+
+  constructor() {
+    super(...arguments);
+    this.record = this.args.person || this.store.createRecord("person");
+    addObserver(this, "args.person", this.personChanged);
+  }
+
+  @action
+  submit(person) {
+    let attributes = [
+      person.languageSkills.toArray(),
+      person.personRoles.toArray()
+    ].flat(1);
+    let picturePath = person.picturePath;
+    let create = this.record.id;
+    super.submit(person).then(
+      r =>
+        r &&
+        super
+          .submit(attributes)
+          .then(r => r && this.savePicture(picturePath))
+          .then(r => r && this.afterSuccess(create))
+    );
+  }
```

Comment:
what does prettier say here :grin:

Reply:
I would add another line between the function, seems weird that prettier says this is fine :thinking:

Reply:
Prettier wants it exactly like this